assemblies/chap-optaplanner-vaccination.adoc

```diff
@@ -0,0 +1,251 @@
+//:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+:OPTAPLANNER-ENT:
+//Variable for the enterprise version conditional statements.
+
+ifdef::OPTAPLANNER-COMM[]
+:PLANNER: OptaPlanner
+:QUARKUS: Quarkus
+:OPTAPLANNER_JDK_VERSION: 11
+:OPTAPLANNER_MAVEN_VERSION: 3.6
+endif::OPTAPLANNER-COMM[]
+
+[id='chap-optaplanner-vaccination_{context}']
+= {PLANNER} on {QUARKUS}: a vaccination appointment scheduler quick start guide
+
+[role="_abstract"]
+You can use the OptaPlanner vaccination appointment scheduler quick start to develop a vaccination schedule that is both efficient and fair. The vaccination appointment scheduler uses artificial intelligence (AI) to prioritize people and allocate time slots based on multiple constraints and priorities.
+
+.Prerequisites
+
+* OpenJDK {OPTAPLANNER_JDK_VERSION} or later is installed. Red Hat build of Open JDK is available from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required).
+
+* Apache Maven {OPTAPLANNER_MAVEN_VERSION} or higher is installed. Maven is available from the https://maven.apache.org/[Apache Maven Project] website.
+
+* An IDE, such as IntelliJ IDEA, VSCode, Eclipse, or NetBeans is available.
+
+ifdef::OPTAPLANNER-ENT[]
+* You have created a Quakus OptaPlanner project as described in xref:optaplanner-quarkus-con_getting-started-optaplanner[].
```

Comment:
```suggestion
* You have created a Quakus OptaPlanner project described in xref:optaplanner-quarkus-con_getting-started-optaplanner[].
```

Reply:
What do you have against 'as' here? I use that a lot.